### PR TITLE
DTM Contextual Help and Concepts guide

### PR DESCRIPTION
**Pull request by lra-sag**
This includes
Contextual help content
Digital Twin concepts guide

### CODE REVIEW

File: content/dtm/dtm-introduction.md [hunk 1]
```diff
@@ -2,17 +2,26 @@
...
+To create an asset, use **asset types** as the blueprint, which consist of one or more **custom properties** as well as one or more child asset types.  
 
-Once the asset hierarchy is created in the DTM application, the hierarchy can be applied to other applications such as OEE or Machine portal. It would save a lot of time and efforts, as the hierarchy is created only once and can be used directly in other {{< product-c8y-iot >}} hosted applications.
 
-Devices for the DTM application are created using the {{< product-c8y-iot >}} Device Management application.
-For more details on the Device Management application, see [Device Management > Connecting devices > Device registration](/users-guide/device-management/#connecting-devices) in the *User guide*.
+An asset hierarchy consist of a root asset type and all its subsequent child asset types. Each hierarchy level consists of custom properties for the asset, as well as its subassets and devices.
```
Comment (spau6) [inline content/dtm/dtm-introduction.md:17]: Each hierarchy level _may_ consists of custom properties for the asset

File: content/dtm/dtm-introduction.md [hunk 2]
```diff
@@ -2,17 +2,26 @@
...
 
-Devices for the DTM application are created using the {{< product-c8y-iot >}} Device Management application.
-For more details on the Device Management application, see [Device Management > Connecting devices > Device registration](/users-guide/device-management/#connecting-devices) in the *User guide*.
+An asset hierarchy consist of a root asset type and all its subsequent child asset types. Each hierarchy level consists of custom properties for the asset, as well as its subassets and devices. 
 
-Assets are created in the DTM application using asset types as the blueprint for assets. An asset type consists of one or more custom properties and subassets.
 
-The DTM application also provides a localization feature to customize the translation for certain content. See [Localization](/dtm/localization) for more information on this feature.
+To get started, open the **Digital twin library** in the **Configuration** menu in the navigator and create the asset types and custom properties. When you created your blueprints, navigate to **Assets** menu in the navigator and click **Add Asset** on the top right corner to define assets. "
```
Comment (spau6) [inline content/dtm/dtm-introduction.md:20]: **Add Asset** change to **Add asset**

File: content/dtm/dtm-introduction-bundle/concepts-guide.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+* Continuously **collects data** (through sensors, devices, etc).
+* Continuously **connected** to the physical asset, updating itself with any change to the asset’s state, condition, and context 
+* Provides value through **visualization**, **analysis**, **prediction**, and **optimization** 
+
+</div>
+</div>
+
+### Asset Hierarchy
+
+An asset hierarchy is a structure that organizes multiple assets into a hierarchy, allowing for the visualization and management of complex systems.
```
Comment (spau6) [inline content/dtm/dtm-introduction-bundle/concepts-guide.md:32]: Can it be "multiple assets/devices into a hierarchy"

File: content/dtm/dtm-introduction.md (hunk 1 above)
Comment (lra-sag) [reply content/dtm/dtm-introduction.md:17]: @MWindel, please provide your comment on @spau6 's suggestion

File: content/dtm/dtm-introduction-bundle/concepts-guide.md (hunk 3 above)
Comment (lra-sag) [reply content/dtm/dtm-introduction-bundle/concepts-guide.md:32]: @MWindel, please provide your view on @spau6's suggestion

File: content/dtm/dtm-introduction-bundle/concepts-guide.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+1. First identify the key systems, processes, and components to be included in order to define the scope of your asset hierarchy.
+
+2. Based on your definitions create the hierarchy of all digital twins.
+
+![Asset hierarchy windmill](/images/dtm/digital-twin/dtm-asset-hierarchy-windmill.png)
+
+3. Determine the level of detail needed for each system, process, and component in the hierarchy. This helps to determine the level of accuracy and complexity required for each asset type. For example, the tower's height, the length of the blade and so on.
+
+4. To portray this complexity, create a custom property for each component in the **Digital twin library** to be used in the individual asset types.
```
Comment (MWindel) [inline content/dtm/dtm-introduction-bundle/concepts-guide.md:41]: @lra-sag does this refer to custom properties?

@MWindel check process in UI

File: content/dtm/dtm-introduction.md (hunk 1 above)
Comment (MWindel) [reply content/dtm/dtm-introduction.md:17]: @lra-sag @spau6 I have incorporated the changes. Just FYI we do not use "may" in documentation. If absolutely necessary, use "can"

**Comment by MWindel** [comment]
@lra-sag The screenshots are too big. Please resize them, for further information check out our screenshot standards in the How to section of the C8Y IoT & Analytics Documentation channel in teams.

File: content/dtm/dtm-introduction-bundle/concepts-guide.md (hunk 4 above)
Comment (lra-sag) [reply content/dtm/dtm-introduction-bundle/concepts-guide.md:41]: @MWindel yes, it refers to custom properties.

File: content/dtm/dtm-introduction.md (hunk 1 above)
Comment (lra-sag) [reply content/dtm/dtm-introduction.md]: noted.

File: content/dtm/dtm-introduction.md (hunk 2 above)
Comment (lra-sag) [reply content/dtm/dtm-introduction.md]: Resolving conversation as @MWindel has already made the changes

File: content/dtm/asset-types-bundle/asset-types.md [hunk 5]
```diff
@@ -2,9 +2,17 @@
 weight: 20
 title: Asset types
 layout: redirect
+helpcontent:
+- label: asset-types
+  title: Asset Types
+  content: "An asset type is a blueprint or template to create one or multiple assets. It defines how an asset will look like after it is created. An asset type consists of one or more custom properties or child asset types.
+
```
Comment (BeateRixen) [inline content/dtm/asset-types-bundle/asset-types.md:9]: As far as I know you have to have two empty rows in the help content in the front matter. Please confirm @carlosceia.

File: content/dtm/dtm-introduction-bundle/concepts-guide.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+---
+weight: 10
+title: Concepts Guide
```
Comment (BeateRixen) [inline content/dtm/dtm-introduction-bundle/concepts-guide.md:3]: Please remove the "Guide" here. This is just the "Concepts" section in the DTM guide, isn´t it? We already have a Concepts guide in the documentation and we do not have guides within guides.

**Comment by lra-sag** [comment]
> @lra-sag The screenshots are too big. Please resize them, for further information check out our screenshot standards in the How to section of the C8Y IoT & Analytics Documentation channel in teams.

@MWindel , when you say screenshots are too big, do you mean the file size or the dimensions? The file sizes are similar to other screenshots in DTM and within the recommended dimensions.
These are the details of the 3 images:
dtm-asset-hierarchy-windmill.png : 830 X 1324, 72.1 kB
dtm-digital-twin.png: 308 X 303, 81.8 kB
dtm-digital-twin-windmill.png: 547 X 637, 36 kB

I can reduce the height of the dtm-asset-hierarchy-windmill.png to 900. But do I need to reduce the size of the other two further? Do recommend a suitable size and I'll incorporate the same.

File: content/dtm/dtm-introduction-bundle/concepts-guide.md (hunk 6 above)
Comment (MWindel) [reply content/dtm/dtm-introduction-bundle/concepts-guide.md:3]: @BeateRixen Yes, that is my understanding.

@lra-sag please verify the matter further.

**Comment by MWindel** [comment]
@lra-sag regarding the screenshot size, the dimensions are too large.

File: content/dtm/asset-types-bundle/asset-types.md (hunk 5 above)
Comment (lra-sag) [reply content/dtm/asset-types-bundle/asset-types.md:9]: The two line breaks were changed to single line break as part of commit 954a79071. Added them back now.

File: content/dtm/dtm-introduction-bundle/concepts-guide.md (hunk 6 above)
Comment (lra-sag) [reply content/dtm/dtm-introduction-bundle/concepts-guide.md]: Incorporated the suggested change.

**Comment by lra-sag** [comment]
> @lra-sag regarding the screenshot size, the dimensions are too large.

@MWindel , I have reduced the dimensions of the images. Please review.

**Comment by MWindel** [comment]
@lra-sag Thank you that looks better. Could you delete static/images/dtm/digital-twin/dtm-digital-twin.png since it won't be needed?
That would be great.

**Comment by lra-sag** [comment]
> @lra-sag Thank you that looks better. Could you delete static/images/dtm/digital-twin/dtm-digital-twin.png since it won't be needed? That would be great.

removed the image as suggested

File: content/dtm/dtm-introduction-bundle/concepts-guide.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+![Asset hierarchy windmill](/images/dtm/digital-twin/dtm-asset-hierarchy-windmill.png)
+
+3. Determine the level of detail needed for each system, process, and component in the hierarchy. This helps to determine the level of accuracy and complexity required for each asset type. For example, the tower's height, the length of the blade and so on.
+
+4. To portray this complexity, create a custom property for each component in the **Digital twin library** to be used in the individual asset types.
+
+5. To create an asset type you must work from the inside out, that is, from the smallest component to the largest. For example, start with the asset type "blade", followed by "rotor", "tower", and "nacelle" and finally "windmill". Assign the required custom property to each asset type as well as the allowed contained asset types (for example, a windmill requires a rotor, which in turn requires rotorblades).
+
+6. Afterwards, add the necessary asset instances. Note that this process starts from the outer most level (for example, the windmill) and ends at the deepest level (for example, the blades).
```
Comment (MWindel) [inline content/dtm/dtm-introduction-bundle/concepts-guide.md:45]:
```suggestion
6. Afterwards, add the necessary asset instances. This process starts from the outer most level (for example, the windmill) and ends at the deepest level (for example, the blades).
```

File: content/dtm/localization.md [hunk 8]
```diff
@@ -2,6 +2,23 @@
...
+  Use the **Localization** feature to add translations for custom properties or asset types, as well as to add custom translations for the existing static text in the DTM application.
+
+
+  Once you created all custom properties and asset types, their names are added as keys in the **Localization** page. You can add further translations as required.  
+
+
+  To add a new translation, click **Add term** on the top menu bar. In the resulting dialog box, add the name of the key, followed by the translation in the respective field.
+
+
+  Note that to use this feature, you must be subscribed to Branding feature application in the Enterprise tenant."
```
Comment (spau6) [inline content/dtm/localization.md:21]: Branding feature is actually named as "Feature-branding"

Comment (lra-sag) [reply content/dtm/localization.md:21]: @MWindel , the initial commit had the right name "Feature-branding" and is changed now.

File: content/dtm/localization.md [hunk 9]
```diff
@@ -16,7 +33,7 @@ title: Localization
 <a name=""></a>
 ### Localization feature
 
-The **Localization** feature is provided in the DTM application. To use this feature, subscribe to [Feature branding](/users-guide/enterprise-tenant/#branding) in the [Enterprise tenant](/users-guide/enterprise-tenant/#overview). Once you have enabled it, open the **Localization** page in the navigator.
+The **Localization** feature is provided in the DTM application. To use this feature, subscribe to [Branding feature](/users-guide/enterprise-tenant/#branding) in the [Enterprise tenant](/users-guide/enterprise-tenant/#overview). Once you have enabled it, open the **Localization** page in the navigator.
```
Comment (lra-sag) [inline content/dtm/localization.md:36]: name of the application to subscribe is Feature-branding

File: content/dtm/localization.md (hunk 8 above)
Comment (lra-sag) [reply content/dtm/localization.md:21]: Resolved it after discussion

**Review by MWindel — APPROVED** [review]

**Review by spau6 — APPROVED** [review]

**Pull request merged**